src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -91,7 +101,10 @@ private string GetResultsFilePath(AssemblyInfo assemblyInfo)
                     lowPriority: false,
                     displayWindow: false,
                     captureOutput: true,
-                    cancellationToken: cancellationToken);
+                    cancellationToken: cancellationToken,
+                    environmentVariables: environmentVariables,
+                    onProcessStartHandler: (process) => ProcDumpRunner.StartProcDump(process.Id, process.ProcessName, outputDirectory, Logger.Log)
```

Comment:
> We use -w option in procdump. This means we wait for the process to start. 

I'm describing the opposite problem. The process starts and completes and then the procdump process starts. Hence it's starting waiting for a process that will never exist (as far as it can see)